### DISCUSSION TITLE
Bootstrap 5 update to its latest version

### DISCUSSION THREAD

**Discussion by peter-menyes** [holger1411/understrap#1880] [Q&A] [answered]
Do you think it's possible to update Bootstrap 5 to the latest version without breaking anything?
(Currently it's Understrap 1.1.0 with Bootstrap v5.1.1 - and 5.2.0 is available.)
If so then replacing the source files could be enough?

**Comment by IanDelMar** [comment, marked as answer]
See https://github.com/understrap/understrap/discussions/1853 for an answer.

**Reply by peter-menyes** [reply]
Thank you, IanDelMar, it's really useful.

(Apologies for the duplicate, I did search but didn't find this.)
